File: Programming-languages-and-laboratory/Python/Exercises/09_compito_java_20160615/Giocatore.py
class Giocatore:
    def __init__(self, codice_squadra, cognome, eta, maglia, ruolo, titolare):
        self._codice_squadra = codice_squadra
        self._cognome = cognome
        self._eta = eta
        self._maglia = maglia
        self._ruolo = ruolo
        self._titolare = titolare
        
    def get_codice(self):
        return self._codice_squadra

    def __str__(self):
        return self._cognome + " " + str(self._eta) + " " + str(self._maglia) + " " + self._ruolo + " " + str(self._titolare)

File: II - Linguaggi di programmazione e laboratorio/Java/Exercises/35_compito_java_2019_11_20/Giocatore.java


public class Giocatore {

    protected int codiceSquadra;
    protected String cognome, nome;
    protected int eta, numeroMaglia;
    protected String ruolo;
    protected boolean titolare;

    public Giocatore(int codiceSquadra, String cognome, String nome, int eta, int numeroMaglia, String ruolo, boolean titolare) {
        this.codiceSquadra = codiceSquadra;
        this.cognome = cognome;
        this.nome = nome;
        this.eta = eta;
        this.numeroMaglia = numeroMaglia;
        this.ruolo = ruolo;
        this.titolare = titolare;
    }

    public String toString() {
        return nome + "\t" + cognome + "\t" + eta + "\t" + numeroMaglia + "\t" + ruolo + "\t" + titolare + "\t";
    }
}
